### PR TITLE
Project config - Recaptcha config

### PR DESCRIPTION
**Pull request by Xiaoshouzi-gh**
Recaptcha config changes in project config.
    - Implemented getProjectConfig.
    - Implemented updateProjectConfig.
    - Updated error code.

### Discussion

  * Read the contribution guidelines (CONTRIBUTING.md).
  * If this has been discussed in an issue, make sure to link to the issue here. 
    If not, go file an issue about this **before creating a pull request** to discuss.

### Testing

  * Make sure all existing tests in the repository pass after your change.
  * If you fixed a bug or added a feature, add a new test to cover your code.

### API Changes

  * At this time we cannot accept changes that affect the public API.  If you'd like to help 
    us make Firebase APIs better, please propose your change in an issue so that we 
    can discuss it together.

### CODE REVIEW

File: src/auth/tenant.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export interface UpdateTenantRequest {
 
   /**
    * The recaptcha configuration to update on the tenant.
+   * By enabling reCAPTCHA Enterprise Integration you are agreeing to reCAPTCHA Enterprise Term of Service.
```
Comment (renkelvin) [inline src/auth/tenant.ts:60]: Is there a link for the ToS?

File: src/auth/index.ts [hunk 2]
```diff
@@ -122,6 +122,15 @@ export {
   TenantManager,
 } from './tenant-manager';
 
+export {
+  UpdateProjectConfigRequest,
```
Comment (renkelvin) [inline src/auth/index.ts:126]: Should we expose GetProjectConfigRequest here?

File: test/unit/auth/project-config-manager.spec.ts [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  before(() => {
+    mockApp = mocks.app();
+    projectConfigManager = new ProjectConfigManager(mockApp);
+    nullAccessTokenProjectConfigManager = new ProjectConfigManager(
+      mocks.appReturningNullAccessToken());
+    malformedAccessTokenProjectConfigManager = new ProjectConfigManager(
+      mocks.appReturningMalformedAccessToken());
+    rejectedPromiseAccessTokenProjectConfigManager = new ProjectConfigManager(
+      mocks.appRejectedWhileFetchingAccessToken());
+
```
Comment (lahirumaramba) [inline test/unit/auth/project-config-manager.spec.ts:71]: nit: extra line

**Review by lahirumaramba — APPROVED** [review]
Thanks, Liubin! LGTM!

File: src/auth/index.ts (hunk 2 above)
Comment (Xiaoshouzi-gh) [reply src/auth/index.ts:126]: We don't have a getProjectConfigRequest.  It takes an empty object.

File: src/auth/tenant.ts (hunk 1 above)
Comment (Xiaoshouzi-gh) [reply src/auth/tenant.ts]: Updated

**Pull request merged**